specs/ClearBrowsingData.md

```diff
@@ -247,22 +254,24 @@ namespace Microsoft.Web.WebView2.Core
 {
     [Flags] enum CoreWebView2BrowsingDataKinds
     {
-        AppCache = 1,
-        FileSystems = 2,
-        IndexedDb = 4,
-        LocalStorage = 8,
-        WebSql = 16,
-        CacheStorage = 32,
-        AllDomStorage = 64,
-        Cookies = 128,
-        AllSite = 256,
-        DiskCache = 512,
-        DownloadHistory = 1024,
-        GeneralAutofill = 2048,
-        PasswordAutosave = 4096,
-        BrowsingHistory = 8192,
-        Settings = 16384,
-        AllProfile = 32768,
+        /// [ObsoleteAttribute("This property is obsolete. Do not use it any more.", true)]
```

Comment:
Please either leave out documentation from this MIDL3 section or you can duplicate the documentation from the COM IDL section above. This is MIDL3 which doesn't have an 'ObsoleteAttribute'. 
```suggestion
```